src/EA.Weee.Web/Views/OrganisationRegistration/JoinOrganisation.cshtml

```diff
@@ -1,10 +1,14 @@
+@using MvcNavigationHelpers
 @using EA.Weee.Web.RazorHelpers
+@using EA.Weee.Web.Controllers
 @model EA.Weee.Web.ViewModels.OrganisationRegistration.JoinOrganisationViewModel
 @{
     ViewBag.Title = "Access Organisation";
 }
+@section full_main_content 
+{
```

Comment:
@stevelee-civica  I dont think this will work. We need a way of determining where we have come from.
You can get here...
1. Login -> Access another org -> find existing org -> select -> continue , click back. We shouldn't go to organisationFound as its the search results screen that would have been the previous page in this scenario.  - This fails
2. Login -> Access another org -> Create small producer -> .... -> find company -> request to join -> back (This works ok)

Reply:
@stevelee-civica  Actually looking at the AC im not sure this change is required at all. 
UJ1: Your organisation has been found
Add back button to take users back to Organisation details

Commented on the other. So you might be able to remove this change completely.

Reply:
removed